### ISSUE TITLE
Arka Planı Hareketli Yapabilir Miyiz

### ISSUE THREAD

**Issue by Fatihyardimci** [Fatihyardimci/ilkders#1]
Web Sitesinin Arka Planı Hareketli Olmasını İstiyorum

**Comment by Fatihyardimci** [comment]
https://codepen.io/leemark/pen/dJxHp Bunu Nasıl Yapıcağım

**Comment by mehmetdavut** [comment]
@hootka bir öneride bulunabilir misin? :)

**Comment by merdemkoc** [comment]
@hootka evet, ben okulda sınav döneminde olduğumdan ilgilenemedim. :( Bu seferlik sen yardımcı olabilir misin?

**Comment by birol** [comment]
@Fatihyardimci Fatih, ben daha yeni gördüm sorunu. Bahsettiğin arkaplanı yapabildin mi?
